health/health_json.c

```diff
@@ -76,7 +82,21 @@ void health_alarm_entry2json_nolock(BUFFER *wb, ALARM_ENTRY *ae, RRDHOST *host)
                    , (ae->flags & HEALTH_ENTRY_FLAG_SILENCED)?"true":"false"
     );
 
-    health_string2json(wb, "\t\t", "info", ae->info?ae->info:"", ",\n");
+    char *replaced_info = NULL;
+    if (likely(ae->info)) {
+        char *m=NULL;
+        replaced_info = strdupz(ae->info);
+        size_t pos = 0;
+        while ( m = strstr (replaced_info+pos, "$family") ) {
+            char *buf=NULL;
+            pos = m - replaced_info;
+            buf = find_and_replace(replaced_info, "$family", ae->family, m);
+            freez(replaced_info); replaced_info = strdupz(buf);
+            freez(buf);
```

Comment:
Please apply `.clang-format` formatting.

Reply:
Thank you ! Applied with https://github.com/netdata/netdata/pull/10961/commits/87dc40f0bc7f52fa2dd3358cef1c2160af90970d